templates/monorepo/packages/react/README.md

```diff
@@ -0,0 +1,5 @@
+# @mono/react
+
+[![Stable release](https://img.shields.io/npm/v/@mono/utils.svg)](https://npm.im/@mono/utils)
+
+Official React SDK for @mono
```

Comment:
It can be either, it’s just a placeholder. I copied it from the project I’m working on. My plan was to make this fairly generic, and then build some more templates/plopfiles for different kinds of packages people will want.